src/coreclr/tools/aot/ILCompiler.Build.Tasks/ComputeManagedAssembliesToCompileToNative.cs

```diff
@@ -99,16 +100,20 @@ public override bool Execute()
             var list = new List<ITaskItem>();
             var assembliesToSkipPublish = new List<ITaskItem>();
             var satelliteAssemblies = new List<ITaskItem>();
-            var nativeAotFrameworkAssembliesToUse = new HashSet<string>();
+            var nativeAotFrameworkAssembliesToUse = new Dictionary<string, ITaskItem>();
```

Comment:
Should this use `OrdinalIgnoreCase` like some of the other path comparisons below? Or are the filenames guaranteed to have some sort of normalized casing?